api/v6/report_server.thrift

```diff
@@ -295,6 +302,14 @@ service codeCheckerDBAccess {
                                    3: Encoding encoding)
                                    throws (1: shared.RequestFailed requestError),
 
+  // Get line content information for multiple files in different positions.
+  // The first key of the map is a file id, the second is a line number:
+  // (e.g.: lineContent = result[fileId][line])
+  // PERMISSION: PRODUCT_ACCESS
+  map<i64, map<i64, string>> getSourceFileLineContent(1: SourceLineList sourceLines,
```

Comment:
The name of this function is really not descriptive. `getLinesInSourceFileContents`?